### DISCUSSION TITLE
Question regarding spotify segment pause and stop function on windows?

### DISCUSSION THREAD

**Discussion by harshv5094** [JanDeDobbeleer/oh-my-posh#3904] [General]
Hello, I have a question regarding Spotify's pause and stop function. I know that it has not been working on my Windows laptop as I read the document. But out of curiosity, I want to ask what are the updates regarding this feature.

**Comment by JanDeDobbeleer** [comment]
@harshv1741 it's never going to work unless Spotify adds that functionality to their title output. That's where we fetch the information from.

**Reply by harshv5094** [reply]
@JanDeDobbeleer So we have to wait for spotify to provide the needed features.

**Reply by JanDeDobbeleer** [reply]
@harshv1741 yes, but I wouldn't count on that.
